packages/env/lib/build-docker-compose-config.js

```diff
@@ -253,7 +283,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 				depends_on: [ 'wordpress' ],
```

Comment:
Do you think it could be problematic that the cli container doesn't use the Dockerfile, but wordpress* does?

Reply:
No, the only thing we're really getting out of the Dockerfile is XDebug support and now the user creation. The impact here is if you run `ls -la` inside of the `cli`'s shell you'll see the `uid` as an owner instead of a specific ID. This doesn't feel important enough to generate a second Dockerfile.